scripts/cnode-helper-scripts/blockPerf.sh

```diff
@@ -13,13 +13,15 @@
 #SELFISH_MODE='Y'      # in case you don't want to share this node's block propagation data, turn the selfish mode on (Y)
 #SERVICE_MODE='N'      # if you deploy-as-service this script it will run with the -s (service) parameter, and surpress console/syslog output. you can overwrite it here, restart the service and watch the console output with 'journalctl -f -u cnode-tu-blockperf'
 
+#CNODE_PORT=6010       # the port on which this node runs (automatically read from cnTools:env   outside cnTools you need to manually set this parameter
```

Comment:
Best to keep default value in [sync](https://github.com/cardano-community/guild-operators/blob/setup-upds/scripts/cnode-helper-scripts/env#L14) with env

Reply:
it comes from `env`  ...ideally. 
This is only for users running the script outside cntools.

Reply:
yip, but default values (even if commented) should be same IMO

Reply:
right, agreed. will become part of next commit.